app/workers/pubsubhubbub/distribution_worker.rb

```diff
@@ -12,10 +12,10 @@ def perform(stream_entry_id)
 
     account = stream_entry.account
     payload = AtomSerializer.render(AtomSerializer.new.feed(account, [stream_entry]))
-    domains = account.followers_domains
+    domains = account.followers_subscription_domains if stream_entry.status&.hidden?
 
     Subscription.where(account: account).active.select('id, callback_url').find_each do |subscription|
-      next unless domains.include?(Addressable::URI.parse(subscription.callback_url).host)
+      next unless !stream_entry.status&.hidden? || domains.include?(Addressable::URI.parse(subscription.callback_url).host)
```

Comment:
This becomes hard to read because of "unless not" so maybe "next if hidden and not included"